_posts/2023-07-05-lts-releases.adoc

```diff
@@ -0,0 +1,47 @@
+
+. *Stability*: LTS versions will be supported for a period of 12 months, allowing users to maintain the same version of Quarkus without worrying about new features introducing regressions or breaking changes. We aim to release a new LTS version every 6 months.
+
+. *Long-Term Support*: LTS versions will continue to receive critical bug fixes, security patches, and performance enhancements even after the introduction of newer versions. Critical issues that directly impact functionality, performance, or security will be prioritized for fixes.
+
+. *Security*: Regular security updates will be part of LTS releases, ensuring your applications stay protected against known vulnerabilities.
+
+. *Identification*: LTS releases will for now be identifed on https://quarkus.io/security. We are working on adding a new badge to the Quarkus website to make it easier to identify LTS releases.
+
+. *Implications for Non-LTS Releases*: Our regular monthly'ish release cycle will continue as before, providing feature updates and improvements at a high pace for those who prefer to stay on the cutting edge.
+
+## Quarkus Extension Maintainers and Contributors
+
+All of the above applies to Quarkus extension maintainers and contributors as well. In addition, we will be recommending that extension maintainers and contributors considers bug fixes and enhancements for LTS releases. This will ensure that LTS releases are as stable and robust as possible.
+
+This means that extension maintainers and contributors will need to consider having branches and versioning in place for LTS releases. 
+
+This is something many extensions already have done in context of Quarkus 2 to 3 move. For example neo4j have a 1.x for Quarkus 2 and 2.x branch for Quarkus 3. Going forward we will recommend that extensions have branch for the LTS version (currently 3.2.x) and one for main Quarkus 3 features, a 3.x.
+
+## Feedback and Help
+
+We're excited to introduce the notion of LTS releases for Quarkus and look forward to your feedback. We probably don't have everything considered or documented yet. If you have any questions or need help, please reach out to us on our [mailing list](https://groups.google.com/g/quarkus-dev) or [chat](https://quarkusio.zulipchat.com/).
+
+We believe this approach will allow us to continue to innovate and deliver new features at a high pace, while also providing users with the stability and predictability they need.
+
+As always, we extend our deepest gratitude to all community members wether users or contributors for your feedback and support. We encourage you to explore the new LTS releases and share your thoughts and experiences with us so we can continue to refine and enhance our offerings based on your input.
+
+Have fun!
```

Comment:
```suggestion
Happy coding!
```

Sorry, but a post about LTS is not about "having fun"

Reply:
its more about having  fun than coding :)